app/Http/Controllers/LessonController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Lesson;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class LessonController extends Controller
+{
+
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    public function index()
+    {
+        $lessons = Lesson::where('user_id', auth()->id())->get();
```

Comment:
You can make this shorten by just calling `Auth::user()->lessons` and it outputs the same with this one.

Reply:
since lessons has relationship with user and we declared that one with our `User` model we can access directly to Lesson object by calling `lessons()`.

Reply:
Resolved in 0f64fad